SYCL/ESIMD/api/functional/type_coverage.hpp

```diff
@@ -282,13 +282,23 @@ template <tested_types required> auto get_tested_types() {
   }
 }
 
-// Syntax sugar to retrieve simd vector dimensions in a consistent way
-template <int... Values> auto inline get_dimensions() {
+// Syntax sugar to retrieve simd vector sizes in a consistent way
+template <int... Values> auto inline get_sizes() {
   return integer_pack<Values...>::generate_unnamed();
 }
 
 // Factory method to retrieve pre-defined values_pack, to have the same
-// default dimensions over the tests
-auto inline get_all_dimensions() { return get_dimensions<1, 8, 16, 32>(); }
+// default sizes over the tests
+auto inline get_all_sizes() { return get_sizes<1, 8, 16, 32>(); }
+
+// It's a deprecated function and it exists only for backward compatibility and
+// it should be deleted in the future.
+auto inline get_all_dimensions() { return get_all_sizes(); }
```

Comment:
```suggestion
// It's a deprecated function and it exists only for backward compatibility and
// it should be deleted in the future. Use get_all_sizes() instead
auto inline get_all_dimensions() { return get_all_sizes(); }
```

Reply:
I have applied this comment in this commit: https://github.com/intel/llvm-test-suite/pull/803/commits/f9168ed3dc8b995f5ed4f7f35384cb2d9719bfb9